docker/base/hacks/usecase/network.disco.make

```diff
@@ -46,7 +46,7 @@ help:
 up:
 	# NB: just specifying the minimal set .. taking advantage of embedded dependencies to bring
 	#     up the remaining containers
-	OK_TESTS="DISABLE_LOGIN" docker-compose up -d mininet floodlight storm-supervisor \
+	docker-compose up -d mininet floodlight storm-supervisor \
```

Comment:
We don't have mininet and topology-engine... Is this `network.disco.make` actual at al?

Reply:
looks like it's not actual > 1 year :(